src/gulp-jsdoc.js

```diff
@@ -81,8 +89,6 @@ export function jsdoc(config, done) {
                     args = args.concat(['-t', inkdocstrap]);
                 }
 
-                args = args.concat(files);
```

Comment:
This doesn't make sense to me. The goal of this change is *not* to do this. Based on your comment above, it seems like the intent is that if the user specifies the `source` object in their config, then we should take the `gulp.src` values and supply them via command-line. I think that is a surprising behavior.

These options should be mutually exclusive in order to prevent strange behavior like files getting documented twice and the `ENAMETOOLONG` error that was originally reported.